docs/dam/ibexa_dam.md

```diff
@@ -0,0 +1,83 @@
+---
+description: DAM module interface for uploading and managing image assets.
+edition: headless
+---
+
+# Ibexa DAM
+
+Digital Asset Management is a platform dedicated to editors. It enables storing in central location, organizing, distributing, 
+and sharing media assets across many channels.
+
+[[= product_name_dam =]] image picker is developed to work with any web browser that supports modern standards. However for the best user experience in editorial work
+it's recommended to use 1366x768 screen resolution.
+List of supported web browsers:
+
+- Mozilla® Firefox® most recent stable version (recommended)
+- Google Chrome™ most recent stable version (recommended)
+- Chromium™ based browsers such as Microsoft® Edge® and Opera®, most recent stable version, desktop and tablet
+- Apple® Safari® most recent stable version, desktop and tablet
+
+## View modes
+
+The main window is available in two display versions: grid and list view.
+
+To change the view mode, in the upper-right corner, click **View** and select the view you need.
+
+### Grid view
+
+Grid view contains image preview, file format, and image size.
+
+### List view
+
+List view contains a thumbnail, title, format, size, dimensions, creation, and update date.
+
+### Sorting
+
+You can sort image assets by alphabetical order or by the creation date.
+To sort image assets, in the upper-right corner, click **Date** and select the
+sorting method you need.
+
+## Search assets
+
+### Keyword search
+
+The search option allows to find image assets using a word or phrase.
+To use it, in the upper part of the screen, click the
+**Search** field. Enter a search keyword. The platform searches assets for matches
+based on the keyword, including file title.
+
+### Filter by attributes
+
+With filtering by attributes, you can narrow search results to the following attributes:
+
+- language
+- format
+- file size
+- orientation
+- dimensions
+- tags (this filter is available only when image assets have tags assigned to)
+- date of creation
+
+To use it, in the right-side panel select attributes and click the **Apply** button.
+You can combine searches by all types of filtering: keyword search and attributes filter.
```

Comment:
We should add very important information that image filtering works only when DXP uses `solr` or `elasticsearch` search engine. LSE (Legacy Search Engine) is not supported.